dev-util/pkgconf/pkgconf-1.5.3.recipe

```diff
@@ -6,28 +6,44 @@ COPYRIGHT="2011-2018 pkgconf authors"
 LICENSE="ISC"
 REVISION="1"
 SOURCE_URI="https://distfiles.dereferenced.org/pkgconf/pkgconf-$portVersion.tar.xz"
-CHECKSUM_SHA256="bab39371d4ab972be1d539a8b10b6cc21f8eafc97f617102e667e82bd32eb234"
-PATCHES="pkgconf-$portVersion.patchset"
+CHECKSUM_SHA256="d3468308553c94389dadfd10c4d1067269052b5364276a9d24a643c88485f715"
+ADDITIONAL_FILES="pkgconf-gcc2.sh"
 
 ARCHITECTURES="!x86_gcc2 ?x86 x86_64"
 SECONDARY_ARCHITECTURES="x86"
 
 commandSuffix="$secondaryArchSuffix"
 if [ "$targetArchitecture" = x86_gcc2 ]; then
 	commandSuffix=""
+	commandBinDir="$prefix/bin"
 fi
 
 libVersion="3.0.0"
 libVersionCompat="$libVersion compat >= ${libVersion%%.*}"
 
+GLOBAL_WRITABLE_FILES="
+	$relativeSettingsDir/pkgconfig/personality.d directory
+	"
+
 PROVIDES="
 	pkgconf$secondaryArchSuffix = $portVersion
+	cmd:pkg_config$secondaryArchSuffix = $portVersion
 	cmd:pkgconf$secondaryArchSuffix = $portVersion
 	lib:libpkgconf$secondaryArchSuffix = $libVersionCompat
 	"
 REQUIRES="
 	haiku$secondaryArchSuffix
 	"
+CONFLICTS="
+	pkgconfig$secondaryArchSuffix
+	"
+
+if [ "$targetArchitecture" = "x86_gcc2" ]; then
```

Comment:
The base package now conflicts with `pkgconfig` directly due to the `pkg.m4` file it shipped with, so I decided to fold `_pkgconfig` subpackage in, since the reason for it to be out is now invalid.

Reply:
> (There will probably be other minor adjustements if we want the recipe to be "`x86_64 hybrid`-ready"

It's already ready :) We only need these scripts for gcc2 as it's not possible to build pkgconf there.

Reply:
What I had in mind when I wrote about the `x86_64 hybrid`-_readiness_ is that we need to make sure that building pkgconf_x86 on x86_64 (when this will be available) will not produce a package that could cause trouble to a primary arch x86_64 build.
I haven't looked close enough, so you might very well be right.
(I might dig this later.)